src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml

```diff
@@ -12,69 +12,132 @@
-                <dt class="govuk-summary-list__key">
+                <dt class="govuk-summary-list__key" id="awarding-organisation-label">
                     @Model.Content.AwardingOrgLabel
                 </dt>
-                <dd class="govuk-summary-list__value" id="awarding-qualification-value">
+                <dd class="govuk-summary-list__value" id="awarding-organisation-value">
                     @Model.AwardingOrganisationTitle
                 </dd>
             </div>
             <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
+                <dt class="govuk-summary-list__key" id="qualification-level-label">
                     @Model.Content.LevelLabel
                 </dt>
                 <dd class="govuk-summary-list__value" id="qualification-level-value">
                     @Model.QualificationLevel
                 </dd>
             </div>
             <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
+                <dt class="govuk-summary-list__key" id="qualification-number-label">
                     @Model.Content.QualificationNumberLabel
                 </dt>
                 <dd class="govuk-summary-list__value" id="qualification-number-value">
                     @Model.QualificationNumber
                 </dd>
             </div>
             <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
+                <dt class="govuk-summary-list__key" id="from-which-year-label">
                     @Model.Content.DateAddedLabel
                 </dt>
                 <dd class="govuk-summary-list__value" id="from-which-year-value">
                     @Model.FromWhichYear
                 </dd>
             </div>
             <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
+                <dt class="govuk-summary-list__key" id="date-of-check-label">
                     @Model.Content.DateOfCheckLabel
                 </dt>
-                <dd class="govuk-summary-list__value">
+                <dd class="govuk-summary-list__value" id="date-of-check-value">
                     @DateTime.UtcNow.ToString("dd MMMM yyyy")
                 </dd>
             </div>
         </dl>
-        <p class="govuk-body" id="additional-requirements-value">@Model.AdditionalRequirements</p>
-        <h2 class="govuk-heading-m">
-            @Model.Content.BookmarkHeading
-        </h2>
-        <p class="govuk-body">
-            @Model.Content.BookmarkText
-            <br>
-            <br>
-            <a href="#" class="govuk-link">@Model.BookmarkUrl</a>
+    </div>
+    <div class="govuk-grid-column-full govuk-!-static-margin-top-6">
+        <h2 class="govuk-heading-l" id="ratio-heading">@Model.Content.RatiosHeading</h2>
+        @Html.Raw(Model.Content.RatiosText)
+        <table class="govuk-table">

```

Comment:
Question: accessibility. Is this table for layout, or is it a genuine table? If it's for layout should we use something other? If it's really a table, then should it have `<th>` headers? Sonar complains on behalf of text readers.

Reply:
This has now been changed to the summary list